Sources/JsonModel/ResultData/ErrorResult.swift

```diff
@@ -2,7 +2,7 @@
 //  ErrorResultObject.swift
 //  
 //
-//  Copyright © 2020 Sage Bionetworks. All rights reserved.
+//  Copyright © 2017-2021 Sage Bionetworks. All rights reserved.
```

Comment:
Is 2017 the actual original date of this file, or should it be 2020 (as in the previous copyright)?

Reply:
This file now includes the *protocol* as well as an object that conforms to that protocol. The protocol was written in 2017.